lib/util/insightspublisher/index.js

```diff
@@ -60,6 +65,12 @@ class InsightsPublisher extends EventEmitter {
       },
       _WebSocket: {
         value: options.WebSocket
+      },
+      _isIpad: {
```

Comment:
@PikaJoyce , For testing, instead of passsing isIpad, isIphone in options, you can mock them using `sinon.stub` 
This lets you stub the whole `../browserdetection` module from the stub. That way you wont have to add these extra member in the class.  You can see an example of this here: https://github.com/twilio/twilio-video.js/blob/76dd81b6fcef180dbf9c030436e8a20a09639ac4/test/unit/spec/signaling/v2/peerconnection.js#L87

Reply:
I'll try this out!